packages/remix-dev/vite/plugin.ts

```diff
@@ -789,7 +793,7 @@ export const remixVitePlugin: RemixVitePlugin = (options = {}) => {
             Array.from(ssrOnlyAssetPaths).map(async (ssrAssetPath) => {
               let src = path.join(serverBuildDir, ssrAssetPath);
               let dest = path.join(assetsBuildDirectory, ssrAssetPath);
-              await fse.move(src, dest);
+              await fse.copy(src, dest);
```

Comment:
For consistency, I also changed from `move` to `copy` so that ssr build assets will be kept intact.